@dlt2018 The build is failing on the AOCODARCF7DUAL target. Please fix
```
/home/runner/work/inav/inav/src/main/target/AOCODARCF7DUAL/target.c:34:23: error: conflicting types for 'timerHardware'
   34 | const timerHardware_t timerHardware[] = {
      |                       ^~~~~~~~~~~~~
In file included from /home/runner/work/inav/inav/src/main/target/AOCODARCF7DUAL/target.c:25:
/home/runner/work/inav/inav/src/main/drivers/timer.h:139:24: note: previous declaration of 'timerHardware' was here
  139 | extern timerHardware_t timerHardware[];
      |                        ^~~~~~~~~~~~~
```

--- a/src/main/target/AOCODARCF7DUAL/target.c
+++ b/src/main/target/AOCODARCF7DUAL/target.c
@@ -31,7 +31,7 @@ BUSDEV_REGISTER_SPI_TAG(busdev_mpu6500,     DEVHW_MPU6500,      MPU6000_SPI_BUS,
 BUSDEV_REGISTER_SPI_TAG(busdev_bmi270,      DEVHW_BMI270,       BMI270_SPI_BUS,     BMI270_CS_PIN,      BMI270_EXTI_PIN,  1,   DEVFLAGS_NONE,  IMU_BMI270_ALIGN);
 
 
-const timerHardware_t timerHardware[] = {
+timerHardware_t timerHardware[] = {
     DEF_TIM(TIM1, CH3, PA10, TIM_USE_PPM, 0, 0),    //PPM
 
     DEF_TIM(TIM8, CH1, PC6, TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR, 0, 1),        // S1   D(2, 2, 7) 

